### PR TITLE
Add Interia RWS record

### PR DESCRIPTION
**Pull request by kubalasecki**
Add Interia related RWS record.

### CODE REVIEW

**Review by shuranhuang — APPROVED** [review]
LGTM

**Comment by CrabbyJim** [comment]
Hello my name is James and I'm not going to stop trying until the end 🔚.
So much you have to do to get together with us to be there for you as soon
as we know what we want. We have to be careful with our clients and I hope
that you will think about this as a sign that maybe you are doing well with
the other things you need to know.Do you want me in this situation. Yes. If
you're interested let them know what you have to be in touch with me to say
that we will have the opportunity to work together with the other guys from
here but also in hopes and prayers in order for those new products you like
to

**Comment by ratchanonsuttawas** [comment]
a20efadfedfaa41d30033bd128ec5ac9f3bbe335

**Review by ratchanonsuttawas — COMMENTED** [review]
related_website_sets.JSON

**Review by ratchanonsuttawas — COMMENTED** [review]
related_website_sets.JSON

**Comment by ratchanonsuttawas** [comment]
> related_website_sets.JSON

**Comment by ratchanonsuttawas** [comment]
> This is necessary verification practices for old version, it is copyright with many different. If I the hacker wanted delete the provas. Make everybody itens on don't have logically, for exemplo curse for dates analyst. And delete other texts importants

**Comment by ratchanonsuttawas** [comment]
> This is necessary verification practices for old version, it is copyright with many different. If I the hacker wanted delete the provas. Make everybody itens on don't have logically, for exemplo curse for dates analyst. And delete other texts importants

**Comment by ratchanonsuttawas** [comment]
> LGTM

**Comment by ratchanonsuttawas** [comment]
> related_website_sets.JSON

**Comment by ratchanonsuttawas** [comment]
> This is necessary verification practices for old version, it is copyright with many different. If I the hacker wanted delete the provas. Make everybody itens on don't have logically, for exemplo curse for dates analyst. And delete other texts importants

>Review these changes

**Review by ratchanonsuttawas — COMMENTED** [review]
>Review these changes 
>> This is necessary verification practices for old version, it is copyright with many different. If I the hacker wanted delete the provas. Make everybody itens on don't have logically, for exemplo curse for dates analyst. And delete other texts importants

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [inline related_website_sets.JSON:1]: > This is necessary verification practices for old version, it is copyright with many different. If I the hacker wanted delete the provas. Make everybody itens on don't have logically, for exemplo curse for dates analyst. And delete other texts importants

**Comment by ratchanonsuttawas** [comment]
> related_website_sets.JSON

**Review by ratchanonsuttawas — COMMENTED** [review]
> related_website_sets.JSON

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > > This is necessary verification practices for old version, it is copyright with many different. If I the hacker wanted delete the provas. Make everybody itens on don't have logically, for exemplo curse for dates analyst. And delete other texts importants

> related_website_sets.JSON

**Review by ratchanonsuttawas — COMMENTED** [review]
+18 -0

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > > > This is necessary verification practices for old version, it is copyright with many different. If I the hacker wanted delete the provas. Make everybody itens on don't have logically, for exemplo curse for dates analyst. And delete other texts importants
> 
> > related_website_sets.JSON

**Comment by BehrangAslani** [comment]
> Add Interia related RWS record.
> 

Record of the time

**Pull request merged**